shoppingCart/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="styling.css">
+    <title>Shopping Cart</title>
+</head>
+<body>
+    <h1>Shopping Cart</h1>
+    <table>
+        <thead>
+            <tr class="tHeadRow">
+                <th>Item</th>
+                <th>Quantity</th>
+                 <th>Price</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>Jetta</td>
+                <td>3</td>
+                <td>24000</td>
+             </tr>
+        </tbody>
+        <tfoot>
+            <tr class="tFootRow">
+                <td>Order Total</td>
+                <td></td>
+                <td>72000</td>
```

Comment:
so i know you have added an id for this order total td, but you will want to add something in your render line item to get the totals to render vs. the static structure you have now